src/mqtt-broker/src/server/grpc/admin/services.rs

```diff
@@ -376,4 +378,43 @@ impl MqttBrokerAdminService for GrpcAdminServices {
 
         Ok(Response::new(reply))
     }
+
+    async fn mqtt_broker_delete_topic_rewrite_rule(
+        &self,
+        request: Request<DeleteTopicRewriteRuleRequest>,
+    ) -> Result<Response<DeleteTopicRewriteRuleReply>, Status> {
+        let req = request.into_inner();
+        let auth_driver = AuthDriver::new(self.cache_manager.clone(), self.client_pool.clone());
+        let config = broker_mqtt_conf();
+        match auth_driver
```

Comment:
The theme rewriting configuration does not need to be stored via auth_driver, it can be stored directly in the placement center